scripts/map.js

```diff
@@ -54,6 +61,7 @@ var loadZones = function (geojson) {
 
       // On layer click, select town
       layer.on('click', function () {
+        console.log("PP:", pp)
```

Comment:
[q] Is this log supposed to be here? If it is, what is PP?

Reply:
Good catch, just removed it 👍